src/config/config_setup.h

```diff
@@ -662,9 +662,9 @@ class ConfigAutoscanSetup : public ConfigSetup {
     /// \brief Creates an array of AutoscanDirectory objects from a XML nodeset.
     /// \param element starting element of the nodeset.
     /// \param scanmode add only directories with the specified scanmode to the array
-    bool createOptionFromNode(const pugi::xml_node& element, std::shared_ptr<AutoscanList>& result);
+    bool createOptionFromNode(const pugi::xml_node& element, std::vector<AutoscanDirectory> result);
```

Comment:
You need a reference otherwise no result is returned

Reply:
Good spot

Reply:
Yep